spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index Page', type: :system do
```

Comment:
- [ ] All your tests in this file are failing. I think this has to do with the `type: :system` setup. When I changed it to `type: :feature` all the tests passed.

  ![image](https://github.com/Wineshuga/pen-it/assets/81468675/80328ce6-2cd7-4520-98cd-d1c71f474e2b)